shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -15,26 +17,46 @@
 public class AndroidTouchProcessor {
 
   // Must match the PointerChange enum in pointer.dart.
```

Comment:
(Off topic, doesn't need to be resolved): this looks like a use case for some kind of cross language serialized data mechanism like we've been talking about elsewhere.

Reply:
Agreed.  This entire category of use-case would benefit.  AndroidTouchProcessor, AndroidKeyProcessor (maybe), AccessibilityBridge, etc.